### DISCUSSION TITLE
Will other prompt frameworks such as Reflexion and Chain of Hindsight be supported in addition to React in the future

### DISCUSSION THREAD

**Discussion by 401557122** [langgenius/dify#8740] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Will other prompt frameworks such as Reflexion and Chain of Hindsight be supported in addition to React in the future

**Comment by datalee** [comment]
mark
